doc/tasks/readme.md

```diff
@@ -0,0 +1,737 @@
+  - Zustandsmenge > 2: Das Bedienelement kann zwischen mehr als 2 fest definierten Zuständen umgeschaltet werden
+  - Kontinuierlich: Das Bedienelement verhält sich wie ein Schieberegler, bei dem ein Wert zwischen einer oberen und einer unteren
+    Grenze eingestellt werden kann
+- Item: Eine Entität, die entweder eine Position im Dungeon haben kann (also bspw. auf dem Boden liegt), oder in einem
+  Inventar gespeichert sein kann und zwischen diesen beiden Zuständen wechseln kann (das ist bewusst vereinfacht, ohne
+  auf die Feinheiten der Item=Entität Thematik einzugehen)
+- Schriftrolle: Ein Item, welches einen Text anzeigt
+- Craftingcontainer: Ein Container, der zusätzlich die Möglichkeit bietet, die Menge der enthaltenen Items unter Verwendung von
+  Craftingrezepten zu transmutieren
+
+## Spielmechaniken
+
+### Text über GUI-Dialog anzeigen
+
+Das Spiel kann Text über ein GUI-Fenster anzeigen, um bspw. Hinweise an die Studierenden zu geben, oder Aufgabentexte
+zu stellen.
+
+### Einen GUI-Button aktivieren
+
+In einem GUI-Dialog können beschriftete Buttons angezeigt werden, die von Studierenden aktiviert werden können, um
+bspw. eine Antwort auf eine Frage auszuwählen.
+
+### Seiten im GUI-Fenster wechseln
+
+Per GUI-Button (z.B. "Weiter", "Zurück") kann zwischen mehreren Seiten eines GUI-Dialogs gewechselt werden.
+
+### GUI Checkboxen anhaken
+
+In einem GUI-Dialog können beschriftete Checkboxen angezeigt werden. Die Checkboxen können durch Anklicken
+aktiviert (falls inaktiv) und deaktiviert (falls bereits aktiv) werden.
+
+### GUI Text eingeben
+
+In einem GUI-Dialog kann eine Textfeld dargestellt werden, in das Studierende Text eingeben können.
+
+### Aufgabe per Questgeber aktivieren
+
+Studierende können mit einem Questgeber interagieren, wodurch sie eine Aufgabe aktivieren.
+
+### Aufgabe per Questgeber abschließen
+
+Falls eine Aufgabe aktiv ist, können Studierende per Interaktion mit dem entsprechenden Questgeber die Aufgabe abschließen.
+
+### Das Inventar eines Containers öffnen
+
+Per Interaktion kann das Inventar eines Containers geöffnet werden, sodass der Inhalt des Inventars einsehbar ist.
+Per GUI-Button kann das Inventar wieder geschlossen werden.
+
+### Item aus Container-Inventar in Charakter-Inventar transferieren
+
+Bei geöffnetem Container-Inventar kann ein Item aus dem Container-Inventar ausgewählt werden und in das Inventar
+des Spielcharakters transferiert werden. Per GUI-Button können auch alle Items aus dem Container-Inventar in das
+Inventar des Spielcharakters übertragen werden.
+
+### Item aus Charakter-Inventar in Container-Inventar transferieren
+
+Bei geöffnetem Spielcharakter-Inventar kann ein Item aus dem Spielcharakter-Inventar ausgewählt werden und in das Inventar
+eines Containers transferiert werden.
+
+### Ein Item aufheben
+
+Ein Item, welches auf dem Boden des Dungeons liegt, kann vom Spielcharakter aufgesammelt werden, indem der Spielcharakter
+mit der Hitbox des Items kollidiert.
```

Comment:
Reicht das schon oder muss der Spieler dann noch was anklicken oder eine Taste drücken?

Reply:
@AMatutat kann da zum Status Quo sicherlich mehr zu schreiben. Unabhängig vom aktuellen Zustand fänd ich das allerdings eleganter, wenn dafür eine dedizierte Aktion nötig ist.

Reply:
Aktuell reicht eine Kollision, das ganze auf Knopfdruck umzubauen ist aber trivial 
#646 und vermutlich spielerisch schöner

Reply:
Habe den Hinweis auf die Tastatureingabe ergänzt

Reply:
@malt-r Kann man das in diesem Dokument offen lassen und nur schreiben "eine dedizierte Aktion"? Das könnte ja alles sein, auch einfach nur "berühren" per Kollision. 

Ansonsten müssten wir nämlich die Interaktionsformen einmal durchdeklinieren - und zwar für alle Aufgaben. Weiter oben hatten wir das mit "GUI oder Tastatureingabe" oder "Klickaktion oder Tastatureingabe" - aber vielleicht ist das hier für die Spielmechanik gar nicht im Detail so wichtig und man sollte einmal am Anfang der Spielmechanik die Begriffe/Kategorien definieren mit ein paar relevanten Beispielen und hier dann immer nur den Begriff nutzen?